src/Commands/HTTPS/InfoCommand.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\HTTPS;
+
+use Consolidation\OutputFormatters\StructuredData\PropertyList;
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Pantheon\Terminus\Site\SiteAwareInterface;
+use Pantheon\Terminus\Site\SiteAwareTrait;
+
+/**
+ * Class InfoCommand
+ * @package Pantheon\Terminus\Commands\HTTPS
+ */
+class InfoCommand extends TerminusCommand implements SiteAwareInterface
+{
+    use SiteAwareTrait;
+
+    /**
+     * Retrieve HTTPS information about an environment
+     *
+     * @authorize
+     *
+     * @command https:info
+     *
+     * @field-labels
+     *    enabled: Enabled?
+     *    ipv4: IPV4
+     *    ipv6: IPV6
+     * @return PropertyList
+     *
+     * @param string $site_env Site and environment in the form `site-name.env`
+     *
+     * @usage terminus https:info <site>.<env>
+     *    Returns information about the HTTPS status of the <env> environment of <site>
```

Comment:
This should be:

`Displays information about the HTTPS status of <site>'s <env> environment.`